tasks/build_tags.py

```diff
@@ -117,7 +117,7 @@
 SECURITY_AGENT_TAGS = {"netcgo", "secrets", "docker", "containerd", "kubeapiserver", "kubelet", "podman", "zlib"}
 
 # SYSTEM_PROBE_TAGS lists the tags necessary to build system-probe
-SYSTEM_PROBE_TAGS = AGENT_TAGS.union({"clusterchecks", "linux_bpf", "npm"}).difference({"python", "trivy"})
+SYSTEM_PROBE_TAGS = AGENT_TAGS.union({"clusterchecks", "linux_bpf", "npm"}).difference({"python", "trivy", "systemd"})
```

Comment:
Is it required ?

Reply:
yes because now that the system probe imports some logs machinery it fails to compile because of missing libraries in the build images. Since we don't need systemd at all this is easier to simply remove it